tests/Feature/Engine/BasicsTest.php

```diff
@@ -77,4 +80,48 @@ public function testSimpleTransitions()
 
         $this->assertEquals(0, $activity->getTokens($dataStore)->count());
     }
+
+    /**
+     * Tests that a process structure has been configured correctly
+     */
+    public function testProcessConfiguration()
+    {
+        //Create a data store
+        $dataStore = $this->dataStoreRepository->createDataStoreInstance();
+
+        //Load the process
+        $process = $this->createSimpleProcessInstance();
+
+        //Create a process instance with the data store
+        $instance = $this->engine->createExecutionInstance($process, $dataStore);
+
+        //Get references to start event and activity
+        $start = $process->getEvents()->item(0);
+        $activity = $process->getActivities()->item(0);
+        $end = $process->getEvents()->item(1);
+
+        //no tokens are returned from the end event
```

Comment:
Please add a "//Assertion: " prefix to all the assertions comments. Ex.
`//Assertion: no tokens are returned from the end event`